CHANGELOG.md

```diff
@@ -9,11 +9,27 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Update Roslyn version (PR: [#6245](https://github.com/dotnet/vscode-csharp/pull/6245))
```

Comment:
Easier to do this roslyn changelog updates when I actually update roslyn.
Also did the rest so its easier laterwhen I snap